consensus/istanbul/validator/weighted.go

```diff
@@ -705,6 +703,21 @@ func filterPoorValidators(weightedValidators []*weightedValidator, stakingAmount
 		}
 	}
 
+	// when no validator has more than minimum staking amount of KLAYs, all members are in validators
+	if len(newWeightedValidators) <= 0 {
+		// 1. if governance mode is not single,
+		// 2. if governance mode is single and governing node does not have minimum staking amount of KLAYs as well
+		if !isSingleMode || (isSingleMode && uint64(govNodeStaking) < amount) {
```

Comment:
@jeongkyun-oh I think this can be simplified like below. What do you think?
```suggestion
		if !isSingleMode || uint64(govNodeStaking) < amount {
```

Reply:
@yoomee1313 It can be. Thanks :)